.changeset/calm-spiders-beg.md

```diff
@@ -0,0 +1,6 @@
+---
+"@medusajs/utils": patch
+"@medusajs/modules-sdk": patch
+---
+
+feat(utils): Remote query object to string array and fields and relations from GQL schema
```

Comment:
This is a bit hard to understand, don't know if my suggestion is much better though 😂 
```suggestion
feat(utils): Convert remote query object to string array of fields and relations from GQL schema
```